requests_toolbelt/utils/deprecated.py

```diff
@@ -32,7 +32,8 @@ def get_encodings_from_content(content):
     :type content: bytes
```

Comment:
We should instead just document here that we're returning a list of bytes objects. For example:

```rst
:returns: Detected encodings in the content as byte-strings.
:rtype: list(bytes)
```

On Python 2 ```str is bytes``` so it's accurate, while on Python 3, ```bytes is bytes``` so the documentation will be more accurate.

I realize that we accidentally changed the return type on Python 3 for this function, but if we accept this change, we'll do the opposite on Python 2 by returning unicode there.